river/drift/ddm.py

```diff
@@ -133,7 +133,7 @@ def update(self, value):
         self._in_warning_zone = False
         self.delay = 0
 
-        if self.sample_count < self.min_num_instances:
+        if self.sample_count <= self.min_num_instances:
```

Comment:
The current implementation is correct. DDM skips the drift detection until it has "seen" at least `min_num_instances`. This change is not necessary.

Reply:
Sorry for the delay. You are right. Although I expect the impact on the performance to be minimal, it is indeed different to the paper. Thanks for pointing this out.